doc/manuals/install_guide.rst

```diff
@@ -34,19 +35,15 @@ following set of commands. Keep in mind that if you're not using
 version 3.6.1, you'll need to update the version number to match your
 download.
 
-`cd ~/Downloads`
+.. code::
 
-`tar zxfv cmake-3.6.1.tar.gz`
-
-`cd cmake-3.6.1`
-
-`./bootstrap --system-curl --no-system-libs`
-
-`make`
-
-`sudo make install`
-
-`sudo ln -s /usr/local/bin/cmake /bin/cmake`
+  cd ~/Downloads
+  tar zxfv cmake-3.6.1.tar.gz
+  cd cmake-3.6.1
+  ./bootstrap --system-curl --no-system-libs --prefix=/opt/kitware/cmake/3.6.1
```

Comment:
Just curious, has this been tested?  Does it work if you don't already have a `/opt` or `/opt/kitware` directory on your system?

Reply:
I don't know if I conveniently have such a system, but I tried:

    $ ./bootstrap --system-curl --no-system-libs --prefix=/opt/cmake/foo/bar/make/new/path/test

...which ought to equally pass/fail (I already have `/opt/cmake`, but obviously not the rest), and it was able to install. I take that as evidence that CMake will create the necessary parent directories.